src/Backend/GameLogic/ContractsAPI.ts

```diff
@@ -752,12 +752,14 @@ export class ContractsAPI extends EventEmitter {
     return this.makeCall(this.contract.getGameover);
   }
 
-  public async getWinners(): Promise<string[]> {
-    return this.makeCall(this.contract.getWinners);
+  public async getWinners(): Promise<EthAddress[]> {
```

Comment:
j curious, what does mapping to EthAddress enable?

Reply:
its just proper typing, we want these to be addresses, not strings. Twitters weren't showing up properly in the Gameover component because the AccountLabel component requires EthAddress and I was doing a ratchet type cast in the component which wasn't working.